DisCatSharp/Clients/DiscordClient.cs

```diff
@@ -615,6 +633,60 @@ public async Task RemoveGuildApplicationCommandsAsync(ulong guildId)
 	public async Task RemoveGuildApplicationCommandsAsync(DiscordGuild guild)
 		=> await this.RemoveGuildApplicationCommandsAsync(guild.Id).ConfigureAwait(false);
 
+	/// <summary>
+	/// Gets the application emojis.
+	/// <param name="fetch">Whether to ignore the cache. Defaults to false.</param>
+	/// </summary>
+	/// <exception cref="BadRequestException">Thrown when an invalid parameter was provided.</exception>
+	/// <exception cref="ServerErrorException">Thrown when Discord is unable to process the request.</exception>
+	public async Task<IReadOnlyList<DiscordApplicationEmoji>> GetApplicationEmojisAsync(bool fetch = false)
+		=> (fetch ? null : this.InternalGetCachedApplicationEmojis()) ?? await this.ApiClient.GetApplicationEmojisAsync(this.CurrentApplication.Id).ConfigureAwait(false);
```

Comment:
This is rather confusing to read imo. I think we should fall back to how the other if checks are done.

Reply:
This is the usual way for cache-aware methods tho as shorthand, I see no need to change that